lib/taskana-core/src/test/java/acceptance/TaskanaDependencyInjectionExtensionTest.java

```diff
@@ -0,0 +1,185 @@
+package acceptance;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import pro.taskana.TaskanaEngineConfiguration;
+import pro.taskana.classification.api.ClassificationService;
+import pro.taskana.classification.internal.ClassificationServiceImpl;
+import pro.taskana.common.api.JobService;
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.common.api.WorkingDaysToDaysConverter;
+import pro.taskana.common.internal.JobServiceImpl;
+import pro.taskana.common.internal.TaskanaEngineImpl;
+import pro.taskana.monitor.api.MonitorService;
+import pro.taskana.monitor.internal.MonitorServiceImpl;
+import pro.taskana.task.api.TaskService;
+import pro.taskana.task.internal.TaskServiceImpl;
+import pro.taskana.workbasket.api.WorkbasketService;
+import pro.taskana.workbasket.internal.WorkbasketServiceImpl;
+
+@ExtendWith(TaskanaIntegrationTestExtension.class)
+public class TaskanaDependencyInjectionExtensionTest {
```

Comment:
also test dependency injection into methods?

Reply:
All I want to test is that all TASKANA Entities can be resolved properly.
The resolution within the constructor, test method or a lifecycle method is a junit thing and therefore there is not need to test that. See: https://junit.org/junit5/docs/current/user-guide/#extensions-parameter-resolution